arbor/cable_cell_param.cpp

```diff
@@ -64,4 +69,74 @@ cable_cell_local_parameter_set neuron_parameter_defaults = {
     0.01
 };
 
+// Discretization policy implementations:
+
+locset cv_policy_max_extent::cv_boundary_points(const cable_cell& cell) const {
+    const auto& emorph = *cell.morphology();
+    const unsigned nbranch = emorph.morph().num_branches();
```

Comment:
The current `em_morp` vs. `morph` interface can be a bit verbose and confusing, e.g. `emporhp.morph().num_branches()`.
Any ideas how to improve this?

Reply:
We could have `em_morphology` derive from `morphology`, but I don't like the strong coupling that entails. Or have a number of methods that just forward queries through to the `em_morphology`.

Reply:
I also prefer the second option.